metabase/metabase
# Show navbar in data app content view

Adds an app navbar to the data app content view (technically a collection content view). Makes sure only relevant actions are displayed in this case (no "edit page" button as there's no page to edit, no "manage content" actions as a user is already in the content view at this point).

### To Verify

1. New > App
2. Click "..." at the top right, and pick "Manage content"
3. Make sure you can see a collection-like table with a list of questions, pages and models in the app
4. Make sure you can't see the pencil icon in the app navbar
5. Click "..." again, and make sure there's no "Manage content" action anymore
6. Make sure the app is still highlighted in the sidebar
7. Click on the page link / "+" button / "App settings" and make sure they all work well

### Demo

https://user-images.githubusercontent.com/17258145/197582552-bbd646a2-2944-41f3-ab43-485d48866509.mp4
